modules/oe_content_tender/config/install/core.entity_form_display.node.oe_tender.default.yml

```diff
@@ -0,0 +1,265 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.oe_tender.body
+    - field.field.node.oe_tender.oe_departments
+    - field.field.node.oe_tender.oe_documents
+    - field.field.node.oe_tender.oe_publication_date
+    - field.field.node.oe_tender.oe_reference
+    - field.field.node.oe_tender.oe_subject
+    - field.field.node.oe_tender.oe_summary
+    - field.field.node.oe_tender.oe_teaser
+    - field.field.node.oe_tender.oe_tender_deadlines
+    - field.field.node.oe_tender.oe_tender_opening_date
+    - node.type.oe_tender
+  module:
+    - datetime
+    - field_group
+    - link
+    - path
+    - rdf_skos
+    - text
+third_party_settings:
+  field_group:
+    group_tabs:
+      children:
+        - group_basic_information
+        - group_main_content
+        - group_metadata
+      parent_name: ''
+      weight: 0
+      format_type: tabs
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+        direction: vertical
+      label: Tabs
+    group_basic_information:
+      children:
+        - title
+        - oe_content_short_title
+        - oe_subject
+        - oe_teaser
+        - oe_summary
+        - oe_reference
+        - oe_publication_date
+        - oe_tender_opening_date
+        - oe_tender_deadlines
+        - oe_departments
+        - oe_content_content_owner
+      parent_name: group_tabs
+      weight: 20
+      format_type: tab
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+        description: ''
+        formatter: open
+        required_fields: true
+      label: 'Basic Information'
```

Comment:
Please dont add groups here at oe_content level only for fields that belong together in logic, see Projects and Events content type form view

Reply:
Fixed. Groups in the form display are removed